courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>avoid using a single char for variable names (it's hard to search &amp; replace and not very informative). It's better to use the word itself rather than its abbreviation</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>